cypress/integration/pages/onDemandAudio/tests.js

```diff
@@ -11,13 +11,13 @@ import getDataUrl from '../../../support/helpers/getDataUrl';
 
 export default ({ service, pageType, variant, isAmp }) => {
   describe(`Tests for ${service} ${pageType}`, () => {
-    describe('Brand image visible above 400px, not visible below 400px', () => {
+    describe('Brand image visible above 360px, not visible below 360px device width', () => {
       it(`Should display image on default viewport (1000x660))`, () => {
         cy.get('div[data-e2e="on-demand-image"]').find('img');
       });
 
-      it(`Should not display image on iphone-6 screen (375x667)`, () => {
-        cy.viewport('iphone-6');
+      it(`Should not display image below 360px device width`, () => {
```

Comment:
Agree it's cool that Cypress can do the viewport stuff, but i'm not sure we should be testing this. We're essentially testing whether a media query + the css `display: none;` rule works.

I think that so long as the snapshots show that valid CSS is there as expected, then we don't need e2es to test the outcome of that visually.